cli/crates/backend/src/api/consts.rs

```diff
@@ -1,6 +1,27 @@
+use std::sync::OnceLock;
+
 pub const CREDENTIALS_FILE: &str = "credentials.json";
-pub const AUTH_URL: &str = "https://app.grafbase.com/auth/cli";
-pub const API_URL: &str = "https://api.grafbase.com/graphql";
 pub const PACKAGE_JSON: &str = "package.json";
 pub const TAR_CONTENT_TYPE: &str = "application/x-tar";
 pub const GRAFBASE_ACCESS_TOKEN_ENV_VAR: &str = "GRAFBASE_ACCESS_TOKEN";
+
+const API_URL: &str = "https://api.grafbase.com/graphql";
+const AUTH_URL: &str = "https://app.grafbase.com/auth/cli";
+
+pub fn auth_url() -> &'static str {
```

Comment:
perhaps `dashboard_url()` is better? to avoid having one env var for every url that leads to the dashboard.
`GRAFBASE_DASHBOARD_URL`

Reply:
We only use it for authentication.

Reply:
Thinking about this so more, why is the federated graph concerned with an auth url anyway?